YoCode/UICheck.cs

```diff
@@ -1,64 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Text.RegularExpressions;
-
-namespace YoCode
-{
-    internal class UICheck
```

Comment:
I assume you removed this deliberately but it wasn't part of this issue. Maybe I missed a conversation or my memory is failing me? Did we just decide it was replaced by better tests now?